src/vos/vos_dtx.c

```diff
@@ -1832,19 +1843,20 @@ vos_dtx_commit_internal(struct vos_container *cont, struct dtx_id *dtis,
 			rc1 = rc;
 
 		if (dce != NULL) {
-			if (slots == 1)
-				pmem_memcpy_nodrain(dce_df, &dce->dce_base,
-						    sizeof(*dce_df));
-			else
+			if (slots == 1) {
```

Comment:
nit - don't need braces

Reply:
I tend to prefer braces when I've split lines (and checkpatch doesn't complain about such cases).  I actually prefer braces all the time though

Reply:
I actually do too :-) ... I just thought checkpatch would have complained about it.

Reply:
It's one of the tricks of the trade to bypass checkpatch...just make sure you have more than one line in your single statement